### PR TITLE
Use flexible dependencies to numpy and pandas

### PR DESCRIPTION
**Pull request by lboucin**

### CODE REVIEW

**Review by lboucin — COMMENTED** [review]
Change self reviewed.

**Review by RobPasMue — APPROVED** [review]
Just one minor comment. Otherwise, LGTM!

File: pyproject.toml [hunk 1]
```diff
@@ -25,8 +25,8 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 importlib-metadata = {version = "^4.0", python = "<3.8"}
-numpy = "1.23.4"
-pandas = "1.5.1"
+numpy = ">=1.14"
+pandas = ">=1.3.2"
 pywin32 = {version = ">=304", markers = "platform_system == 'Windows'"}
```
[the remark this answers is not in the document]
Comment (lboucin) [reply pyproject.toml:30]: done!

**Review by MaxJPRey — APPROVED** [review]
Thanks @lboucin for the quick modifications.

**Pull request merged**